src/Sylius/Behat/Context/Api/Admin/ManagingOrdersContext.php

```diff
@@ -345,4 +350,140 @@ public function theFirstOrderShouldHaveNumber(string $number)
 
         Assert::same($firstItem['number'], str_replace('#', '', $number));
     }
+
+
+    /**
+     * @When I specify filter date from as :dateTime
+     */
+    public function iSpecifyFilterDateFromAs($dateTime)
```

Comment:
Please use new `Context/Api` as example, it's covered with type hints.

```suggestion
    public function iSpecifyFilterDateFromAs(string $dateTime): void
```